src/app/banking/vouchers/search-redeemed-vouchers.component.ts

```diff
@@ -38,15 +37,14 @@ export class SearchRedeemedVouchersComponent
   }
 
   protected getFormControlNames(): string[] {
-    return ['types', 'periodBegin', 'periodEnd'];
+    return ['types', 'periodBegin', 'periodEnd', 'operator'];
   }
 
   protected toSearchParams(value: any): UserVoucherSearchParams {
-    const params = cloneDeep(value);
+    const params: UserVoucherSearchParams = value;
     params['user'] = this.route.snapshot.paramMap.get('user');
```

Comment:
Better use params.user instead
```suggestion
    params.user = this.route.snapshot.paramMap.get('user');
```